samples/jenkins/scripts/prod/run_model_deployment_validation.sh

```diff
@@ -0,0 +1,37 @@
+export PATH=$PATH:$PWD
+
+results_path = 'prod_job_results.zip'
+
+find_asset () {
+  echo "Searching for $1 with name: $2..." >&2
+  asset_id=$(cpdctl asset search --type-name $1 --query "asset.name:$2" \
+    --output json --jmes-query "results[0].metadata.asset_id" --raw-output)
+  echo "Found: $asset_id" >&2
+  echo $asset_id
+}
+
+prod_space_id=$(cpdctl space list --output json --jmes-query "resources[?entity.name == 'prod-space'] | [0].metadata.id" --raw-output)
+export CPD_SCOPE=cpd://cpd/spaces/$prod_space_id
+
+prod_evaluation_script_id=$(find_asset script "evaluate_model.py")
+
+evaluate_model_job_id=$(cpdctl job list --output json --jmes-query "results[?metadata.name == 'evaluate_model_job'] | [0].metadata.asset_id")
+echo "Run starting for a job: $evaluate_model_job_id..."
+
+cpdctl job run create --job-id "$evaluate_model_job_id" --job-run '{}'
+
+results_asset_id=$(cpdctl asset search --type-name data_asset --query "evaluation_result.zip*" --output json \
```

Comment:
If the result of the evaluation is the .zip archive, it could be also attached to the Jenkins job as an artifact using step:
```
archiveArtifacts artifacts: 'evaluation_result.zip', fingerprint: true
```
The similar approach is done in the `export_dev_space.sh` script.